setup/tinyHIPPOstart

```diff
@@ -10,13 +10,25 @@
 opkg update
 
 # Make sure Python3, Pip, misc packages are installed
-opkg install python3
-opkg install python3-pip
 pip3 install scapy
 pip3 install dataclasses
+pip3 install requests
 
 # set config path environment variable
-export IOT_IDS_CONFIGPATH = "/etc/tinyHIPPO/config.json"
+export IOT_IDS_CONFIGPATH='/etc/tinyHIPPO/config.json'
+
+# Move the dashboard files
+# TODO: this is a hack to get tinyHIPPO running for the midterm progress report
```

Comment:
What would make this not-hacky?

Reply:
Bumping what Bri said here, why is this hacky? Don't you need to copy these files to the necessary path? Or can we modify an environment variable or something that points OpenWrt to the directory where the source code for the luci dashboard is needed?

Reply:
It is hacky because the Makefile and Openwrt Installation *should* take care of all the moving of files. So far, all my attempts to move the GUI related files have been unsuccessful. I should eventually figure it out, just more time is needed.